aidants_connect_web/decorators.py

```diff
@@ -12,9 +12,11 @@ def activity_required(view=None, redirect_field_name="next"):
     """
 
     def test(user):
-        time_since_last_action = timezone.now() - user.get_last_action_timestamp()
-        is_alive = time_since_last_action < settings.ACTIVITY_CHECK_DURATION
-        return is_alive
+        if user.get_last_action_timestamp():
+            time_since_last_action = timezone.now() - user.get_last_action_timestamp()
```

Comment:
```suggestion
            time_since_last_action = timezone.now() - last_action_ts
```

(pas besoin de refaire le même appel)